caffeinated/admin/extend/registration_form/templates/question_groups_main_meta_box.template.php

```diff
@@ -53,8 +53,8 @@ class="regular-text"><br/>
                 </label>
             </th>
             <td>
-                <input id="QSG_identifier" name="QSG_identifier<?php echo $id; ?>"
-                       value="<?php $question_group->f('QSG_identifier') ?>" type="text"
+                <input id="QSG_identifier" name="QSG_identifier<?php echo esc_attr($id); ?>"
```

Comment:
```suggestion
                <input id="QSG_identifier" name="QSG_identifier<?php echo absint($id); ?>"
```
                
FYI, almost all of our models use an integer for the primary key (db autoincrement value) but there are a few exceptions like the country table which uses the country ISO value like `US`, `CA`, `IR` etc